src/cloud/paths/tasks.yml

```diff
@@ -212,7 +212,9 @@ post:
     content:
       application/json:
         schema:
-          $ref: "../schemas/TaskCreateRequest.yml"
+          oneOf:
```

Comment:
@glinton Based on the earlier conversation, it sounds like this may not work? It'd be nice to test it and know for sure.

Reply:
I wasn't as familiar with the times when it doesn't work, but I see the oats test isn't complaining, maybe when it goes to be used, so I'll leave that bit up to consumers of the spec (this is producing valid swagger).